spec/lib/bank_accounts/account_spec.rb

```diff
@@ -0,0 +1,58 @@
+require './spec/spec_helper'
+require './lib/bank_accounts/account'
+
+describe BankAccounts::Account do
+
+
+  describe "class methods" do
```

Comment:
Even when it's still high level? So this would be `describe "account actions"` or something?